include/Dialect/Btor/IR/BtorOps.td

```diff
@@ -945,15 +952,21 @@ def InitArrayOp : Btor_Op<"array"> {
     For example:
 
     ```mlir
-    %4 = btor.array %1 : !btor.array<i8,i32>
+    %4 = btor.array %1 : vector<8,i32>
```

Comment:
`vector<8,i32>` is not same as `array<i8,i32>`, should be `vector<2**8,i32>`

Reply:
They aren't meant to be the same. This is just a new example - `vector<8,i32>` is meant to be the same as `array<i3,i32>`